egs/wsj/s5/utils/reverse_lm.sh

```diff
@@ -38,25 +38,13 @@ mkdir -p $outdir
 for f in phones.txt words.txt L.fst L_disambig.fst phones/; do
   cp -r $langdir/$f $outdir
```

Comment:
>
> In egs/wsj/s5/utils/reverse_lm.sh
> <https://github.com/kaldi-asr/kaldi/pull/682#discussion_r59294521>:
>
> > @@ -38,25 +38,13 @@ mkdir -p $outdir
> >  for f in phones.txt words.txt L.fst L_disambig.fst phones/; do
> >    cp -r $langdir/$f $outdir
>
> The reverse_lm script is used only in aurora4/s5 and sprakbanken/s5. The
> latter is free, so I can probably pull a real grammar. This should be
> thorough enough.
>
> Another interesting question is should we try to implement and maintain
> unit tests for the utility scripts (and executable tools by extension).
> Sounds nasty, but a quick search reveals a few bash unit-testing
> frameworks. The implementation is for another day indeed, as there is just
> one of me, unfortunately. But as a point to be in--does it make sense?
>
A few of the utilities have comments at the end indicating how they were
tested.  Regarding unit-testing the scripts I'm not sure how many of them
would be amenable to unit testing.  Also the line where Kaldi ends and the
rest of the world begins can be a little fuzzy, and the range of
environments where the scripts are supposed to work is not always well
defined.  But at least seeing what it would look like would be interesting.
Dan




> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/682/files/6e8dd7a3e849f08d04b4fb28cd3fbd772297d4bd#r59294521>
>